.gitignore

```diff
@@ -6,4 +6,5 @@ lib/
 out/
 node_modules/
 *.vsix
-*.tsbuildinfo
\ No newline at end of file
+*.tsbuildinfo
+examples/*/example/generated
```

Comment:
The output files now seem to be generated into subfolders named after the source file: `blog` and `qualifiednames`. So this ignore line has no effect.

Please create a new `.gitignore` file in `examples/domainmodel` listing the generated subfolders to be ignored.